### PR TITLE
Fix indent and use println instead of print (`starship init {shellname}`)

### PR DESCRIPTION
**Pull request by kyoheiu**
#### Description

- Use `println!` instead of  `print!` in `src/init/mod.rs` to add a line break properly.
- Remove indents of bash script to show the code properly. (indoc crate seems great to use in this case though)

#### Motivation and Context

1. First, the script printed by `starship init bash` is indented, which is not necessary.

2. Second, every script printed by `starship init {shellname}` does not have new line character at the end of it, which makes the next shell prompt appear without a line break.

Note that the line break issue appears differently according to the terminal emulator we use: For example, alacritty shows this issue, while wezterm does not.

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [x ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

**Comment by davidkna** [comment]
Have you tested your changes? I think some shells have issues evaluating multiple lines at once.

**Comment by kyoheiu** [comment]
I cannot test on cmd and ion, but for other shells:

elvish:
elvish

fish:
fish

nushell:

tcsh:

xonsh:

zsh:

**Comment by kyoheiu** [comment]
(Sorry for unintended close)

I cannot test on cmd and ion, but for other shells:

elvish:
elvish

fish:
fish

nushell:
nushell

tcsh:
tcsh

xonsh:
xonsh

zsh:
zsh

**Comment by davidkna** [comment]
Unless someone objects, I would rather not merge the `println!` changes at this time, because there is little benefit, but a certain risk of something breaking - and testing all combinations of shells would be difficult. If you remove those changes, I wouldn't be opposed to merging the indent change.

**Comment by kyoheiu** [comment]
Fixed to use `print!` instead of `println!`.

File: src/init/mod.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@ pub fn init_stub(shell_name: &str) -> io::Result<()> {
     let starship = StarshipPath::init()?;
 
     match shell_basename {
-        "bash" => print!(
+        "bash" => println!(
```
Comment (davidkna) [inline src/init/mod.rs:104]:
```suggestion
        "bash" => print!(
```

File: src/init/mod.rs [hunk 2]
```diff
@@ -178,7 +178,7 @@ pub fn init_stub(shell_name: &str) -> io::Result<()> {
         ),
         "cmd" => print_script(CMDEXE_INIT, &StarshipPath::init()?.sprint_cmdexe()?),
         _ => {
-            eprintln!(
+            eprint!(
```
Comment (davidkna) [inline src/init/mod.rs:181]:
```suggestion
            eprintln!(
```

File: src/init/mod.rs [hunk 3]
```diff
@@ -229,7 +229,7 @@ pub fn init_main(shell_name: &str) -> io::Result<()> {
 
 fn print_script(script: &str, path: &str) {
     let script = script.replace("::STARSHIP::", path);
-    print!("{}", script);
+    println!("{}", script);
```
Comment (davidkna) [inline src/init/mod.rs:232]: The files typically already have a final newline.
```suggestion
    printl!("{}", script);
```

**Review by davidkna — CHANGES_REQUESTED** [review]

File: src/init/mod.rs (hunk 3 above)
Comment (kyoheiu) [reply src/init/mod.rs]: You mean `print` right? Fixed it in my commit [06e1c62](https://github.com/starship/starship/pull/4429/commits/06e1c6296395a1e3e69e6d646ade59323372ac25)

**Review by davidkna — APPROVED** [review]

**Pull request closed without merging**